### DISCUSSION TITLE
Ajouter une infos dans le wiki côté pet

### DISCUSSION THREAD

**Discussion by FLORIAN4600** [Crownicles/Crownicles#1357] [Suggestions fermées]
Ce serait bien, si on pouvait savoir le nombre de points qu'il faut pour faire level up son pet (au lieu de devoir lire le code de Draftbot pour le savoir)

**Comment by BastLast** [comment]
Tu peux le savoir via le vétérinaire non ?

**Reply by Pietagorh** [reply]
Tu peux savoir combien t'as de points sur 100, mais pas quels sont les paliers

**Reply by BastLast** [reply]
Bah oue logik

**Reply by BastLast** [reply]
C un peu volontaire de cacher cet info

**Reply by FLORIAN4600** [reply]
Ok, merci de vos réponse
